.github/workflows/ci.yaml

```diff
@@ -65,6 +65,9 @@ jobs:
     - uses: actions/checkout@v2
       name: Checkout current branch
 
+    - name: .NET (Core) SDK Information
```

Comment:
Do we want to keep this in the ci, or was it just for debugging?

Reply:
I think it's harmless and worthwhile. Since we allow rolling-forward by feature, it could help if we start seeing different tooling behaviours being observed on CI and development environments. Perhaps then we'll need to pin down more strictly, but which can prove to be cumbersome for developers who may have a more recent version.